.travis.yml

```diff
@@ -1,9 +1,15 @@
 language: python
+sudo: false
+cache:
+  directories:
+    - node_modules
+    - $HOME/virtualenv/python$TRAVIS_PYTHON_VERSION/lib/python$TRAVIS_PYTHON_VERSION/site-packages/
```

Comment:
I tried this. The problem is, that the the .virtualenv directory is very big (100MB) which have to be loaded for any run and have to be compressed and send back after each run. This took so long, that there was no speed gain. See the documentation of travis cache for this.